packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -41,6 +52,13 @@
 	}
 }
 
+// Ensure that the height of the first appender, and the one between blocks, is the same as text.
+.block-editor-block-list__block[data-type="core/paragraph"] p,
```

Comment:
Does this style belong here or in the paragraph block instead?

Reply:
Yes, I think so, I moved the styles to the paragraph block 👍